src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj

```diff
@@ -10,13 +10,14 @@
     <PackageDescription>Default implementation of dependency injection for Microsoft.Extensions.DependencyInjection.</PackageDescription>
     <!-- Use targeting pack references instead of granular ones in the project file. -->
     <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
I wonder if we should update `Microsoft.Extensions.Hosting` as well, so service workers automatically get the new version of DependencyInjection. Thoughts?

Reply:
Honestly I don't know. If you think it is safe to update `Microsoft.Extensions.Hosting`, then we can do it anyway. 

CC @ViktorHofer @ericstj

Reply:
It probably isn't super important for most Hosting apps, and people who need the fix can upgrade themselves.

ASP.NET will automatically get the new version, so it shouldn't be a concern for them.

Reply:
Sounds good then.

Reply:
Hmm, Eric considered adding infrastructure to force libraries that depend on a patched library to ship during servicing as well but he abandoned that idea after talking to various folks on the team. I think it's OK to not update dependent libraries.

Reply:
We discussed this in tactics and folks wish to remain on incremental servicing path for packages.  We aren't explicitly shipping additional packages just for the sake of bringing in transitive references.  Should we ship Hosting for some other reason in the future it will get the updated reference.